src/tests/functional/plugin/gpu/shared_tests_instances/single_layer_tests/softmax.cpp

```diff
@@ -62,9 +63,34 @@ const auto params4D = testing::Combine(
 
 INSTANTIATE_TEST_SUITE_P(
         smoke_SoftMax4D,
-        SoftMaxLayerTest,
+        SoftMax8LayerTest,
         params4D,
-        SoftMaxLayerTest::getTestCaseName
+        SoftMax8LayerTest::getTestCaseName
+);
+
+const std::vector<ov::Shape> inputShapes5D = {
+    {1, 100, 1, 1, 1},
+    {1, 3, 4, 3, 4},
+    {2, 3, 4, 5, 6},
+};
+
+const std::vector<int64_t> axis5D = {-5, -4, -3, -2, -1, 0, 1, 2, 3, 4};
+
+const auto params5D = testing::Combine(
+    testing::ValuesIn(netPrecisions),
```

Comment:
Please add f16  test also. It will be aligned with clDNN_unit_test which has both f16 and f32 test case.

Reply:
Done.

The `abs_threshold` of the test had to be tuned for f16 case -- please have a look at the corresponding TODO in the last commit: https://github.com/openvinotoolkit/openvino/pull/10364/commits/4ebb49c6b00d7e38b6a2d64855658ba6d8429a3c

Reply:
Got it! Understood the change of abs_threshold for fp16 case.